transaction-impl/src/main/java/com/example/auction/transaction/impl/TransactionServiceImpl.java

```diff
@@ -86,6 +88,14 @@ public TransactionServiceImpl(PersistentEntityRegistry registry, ItemService ite
     }
 
     @Override
+    public ServiceCall<Boolean, Done> submitPaymentStatus(UUID itemId) {
+        return authenticated(userId -> request -> {
+            SubmitPaymentStatus submitPaymentStatus = new SubmitPaymentStatus(userId, request);
+            return entityRef(itemId).ask(submitPaymentStatus);
```

Comment:
If request wasn't a `boolean` we would be able to map the values from the request into internal `TransactionStatus` here.